tests/robot-tests/tests/general_public/methodology_page_absence.robot

```diff
@@ -8,7 +8,7 @@ Force Tags          GeneralPublic    Local    Dev    Test    Preprod
 
 *** Test Cases ***
 Navigate to Pupil absence in schools in England methodology page
-    [Tags]    HappyPath
+    [Tags]    HappyPath  Failing  NotAgainstDev
```

Comment:
Rather than adding these tags to every test case, you could add them to the `Force Tags` list at the top once.

Also, `Failing` stops the tests running against all environment, so the `NotAgainstDev` tag is redundant while the `Failing` tag is applied. `NotAgainstDev` is meant to be applied in situations where we don't want the test case to be run against the dev environment specifically.